test/integration/abm_gce_defaults_on_editor_project/abm_gce_editor_test.go

```diff
@@ -51,7 +51,7 @@ func TestABMEditor(t *testing.T) {
 			assert.False(vm.Get("guestAccelerators").Exists(), "guestAccelerators are disabled")
 		}
 
-		abmInstall := runSSHCmd(t, projectID, "tfadmin@cluster1-abm-ws0-001", "sudo ./run_initialization_checks.sh")
+		abmInstall := runSSHCmd(t, projectID, "tfadmin@cluster1-abm-ws0-001", "export GOOGLE_APPLICATION_CREDENTIALS=/home/tfadmin/terraform-sa.json && sudo -E ./run_initialization_checks.sh")
```

Comment:
nit: maybe it worth to place the path to a constant string instead of copy pasting it downstream